README.md

```diff
@@ -56,3 +55,5 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
 
 If you need any help anywhere in the process, you can always ask a question on our [Gitter Chat](https://gitter.im/jboss-outreach/gci).
 
+# Wireframe for student's dashboard
+![alt text](https://user-images.githubusercontent.com/36305142/47714157-1ca0b500-dc62-11e8-894f-a81dd6198111.png)
```

Comment:
Hi, thanks for your efforts. Any views on how to handle more number of certificates and not just those present in the view?

Reply:
Hey, maybe we can restructure the entire dashboard and display a search bar and the total number of certificates issued. So, if a student want to see their certificate, they can just enter their ID and view it. With this method, privacy of other student's certificate can be provided. Just my opinion.

Reply:
It would be great if you include that too in your designs because we'll begin coding for templates in a day or two and hence an elaborate design is much appreciated. Thanks for your inputs

Reply:
Sure! I got that idea when I read this - "handle more number of certificates and not just those present in the view". I will be implementing the design soon.

Reply:
Uploaded... Please check!

Reply:
This is still ambiguous as a design, why not introduce concept of pagination with in view certificates sorted by some order maybe alphabetical or last modified... and say a search box on top so that your idea of searching certificates is retained. 
Please don't forget to squash your commits, a lots of commits are visible in single PR. Make sure your update your fork before making a PR. Thanks

Reply:
I have updated my fork and created a new PR. Closing this PR and updating designs in the new one